base/applications/mspaint/dialogs.cpp

```diff
@@ -74,6 +74,9 @@ LRESULT CMirrorRotateDialog::OnCancel(WORD wNotifyCode, WORD wID, HWND hWndCtl,
 
 LRESULT CMirrorRotateDialog::OnRadioButton3(WORD wNotifyCode, WORD wID, HWND hWndCtl, BOOL& bHandled)
 {
+    if (!IsDlgButtonChecked(IDD_MIRRORROTATERB3) != BST_CHECKED)
```

Comment:
```suggestion
    if (IsDlgButtonChecked(IDD_MIRRORROTATERB3))
```
or just like that?

Reply:
`IsDlgButtonChecked` has a tri-state value. I would use a strict style.